docs/fundamentals/code-analysis/quality-rules/ca2262.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CA2262: Set 'MaxResponseHeadersLength' properly"
+description: "Learn about code analysis rule CA2262 - Set 'MaxResponseHeadersLength' properly"
+ms.date: 01/30/2024
+f1_keywords:
+  - CA2262
+helpviewer_keywords:
+  - CA2262
+dev_langs:
+  - CSharp
+  - VB
+---
+# CA2262: Set 'MaxResponseHeadersLength' properly
+
+| Property                            | Value                                       |
+|-------------------------------------|---------------------------------------------|
+| **Rule ID**                         | CA2262                                      |
+| **Title**                           | Set `MaxResponseHeadersLength` properly     |
+| **Category**                        | [Usage](usage-warnings.md)                  |
+| **Fix is breaking or non-breaking** | Non-breaking                                |
+| **Enabled by default in .NET 9**    | As suggestion                               |
+
+## Cause
+
+The <xref:System.Net.Http.HttpClientHandler.MaxResponseHeadersLength?displayProperty=nameWithType> property is set to a value greater than 128.
+
+## Rule description
+
+The <xref:System.Net.Http.HttpClientHandler.MaxResponseHeadersLength?displayProperty=nameWithType> property is measured in kilobytes, not bytes. The default maximum length is 64 KB. If you set the property to a value greater than 128 kilobytes, it might be due to a misunderstanding of the units of this property.
```

Comment:
NIT: maybe add something like
```suggestion
The <xref:System.Net.Http.HttpClientHandler.MaxResponseHeadersLength?displayProperty=nameWithType> property is measured in kilobytes, not bytes. The default maximum length is 64 KB which should be enough for the vast majority of use case. If you set the property to a value greater than 128 kilobytes, it might be due to a misunderstanding of the units of this property.
```